CONCEPTS.md

```diff
@@ -0,0 +1,62 @@
+# Concepts
+
+## Table of Contents
+- [Metric](#metric)
+- [Tag Key and Tag Value](#tag-key-and-tag-value)
+- [Timestamp](#timestamp)
+- [Timeseries](#timeseries)
+- [Gauge](#gauge)
+- [Counter](#counter)
+- [Timer](#timer)
+- [Metric Registry](#metric-registry)
+- [Reporter](#reporter)
+- [Time Window](#time-window)
+- [Monoid](#monoid)
+
+### Metric
+A quantifiable characteristic of an application that is being measured. Metrics have numeric values that are either 64-bit signed integers or 64-bit double-precision floating-point values. Metrics have names that are strings and should meaningfully describe the quantity being measured. Some examples -
+* `webserver.rps` the number of requests per second being processed by a web server
```

Comment:
I am not sure ‘rps’ is good name, as one could argue ‘requests’ is what you measure, the rate is just one way of querying it.